pages/register.py

```diff
@@ -25,6 +25,11 @@ class Register(Base):
     _recaptcha_checkbox_locator = (By.CSS_SELECTOR, '.recaptcha-checkbox-checkmark')
     _recaptcha_checkbox_checked = (By.CSS_SELECTOR, '.recaptcha-checkbox-checked')
 
+    def __init__(self, base_url, selenium, open_url=True):
+        Base.__init__(self, base_url, selenium)
+        if open_url:
+            self.selenium.get(self.base_url)
+
```

Comment:
Why is this needed, where it wasn't before? This is a practice we've moved away from, which is why I ask.